### PR TITLE
Add French translations

### PR DESCRIPTION
**Pull request by NathanBnm**
Closing #6.

### CODE REVIEW

**Comment by NathanBnm** [comment]
Please note that some strings are missing like 'Select at least one language to generate .gitignore!', 'Selected Languages' and 'Reset selected languages'.

**Comment by arshubham** [comment]
@NathanBnm Those strings were removed in the redesign.

File: po/fr.po [hunk 1]
```diff
@@ -1,131 +1,136 @@
...
-msgstr ""
+msgstr "Refonte de l'interface"
 
 #: data/com.github.arshubham.gitignore.appdata.xml.in:34
 msgid "Added i18n support"
-msgstr ""
+msgstr "Ajout du support i18n"
 
 #: data/com.github.arshubham.gitignore.appdata.xml.in:35
 msgid "Added French Translations"
```
Comment (arshubham) [inline po/fr.po]: Added Just one more string here.

**Comment by NathanBnm** [comment]
@arshubham that's a good idea :wink:

File: po/fr.po (hunk 1 above)
Comment (NathanBnm) [reply po/fr.po]: Just added

**Comment by arshubham** [comment]
Thanks a lot for the pr. Will be making a release today.

**Pull request merged**